vanilla-framework/vanilla-brochure-theme
# Accent colors

## Done

- Update `.sass-lint.yml` to match Vanilla framework
- Change `$color-accent` to match `$color-brand` by default
- Add `$color-accent-background` and make it match  `$color-brand` by default
- Tweak `is-ticked` list styling so it uses `$color-accent` by default

## QA

- Check code
- Run `gulp jekyll`
- Go to http://127.0.0.1:4001/vanilla-brochure-theme/examples/patterns/lists/split-list/
- Go to `_settings.scss` and change value of `$color-accent`
- Check that ticks change accordingly

## Details

Fixes #77
